resolve.js

```diff
@@ -29,7 +29,7 @@ const options = {
     externals: [],
     externalRefs: {},
     rewriteRefs: true,
-    status: 'undefined',
+    status: 'undefined'
```

Comment:
Trailing `,` is fine, otherwise lots of line changes.  Your call.

Reply:
Just following the eslint config that's in the root of the repo!